test/bin.spec.js

```diff
@@ -56,18 +56,19 @@ export const testW3 = {
 }
 
 export const testAccount = {
-  'w3 account ls': test(async (assert, context) => {
-    const { output } = await w3
-      .env(context.env.alice)
-      .args(['account ls'])
-      .join()
+  // 'w3 account ls': test(async (assert, context) => {
+  //   const { output } = await w3
+  //     .env(context.env.alice)
+  //     .args(['account ls'])
+  //     .join()
+  //   console.log('out', output)
 
-    assert.match(output, /has not been authorized yet/)
-  }),
+  //   assert.match(output, /has not been authorized yet/)
+  // }),
 
   'w3 login': test(async (assert, context) => {
     const login = w3
-      .args(['login', 'alice@web.mail'])
+      .args(['login', 'alice-cli-test@web.mail'])
```

Comment:
looks like `alice@web.mail` is flagged as spam email and we cannot use it anymore? I get an error